docker-app/qfieldcloud/core/exceptions.py

```diff
@@ -58,6 +58,14 @@ class NotAuthenticatedError(QFieldCloudException):
     status_code = status.HTTP_401_UNAUTHORIZED
 
 
+class TooManyLoginAttemptsError(QFieldCloudException):
+    """Raised when QFieldCloud had too many failed login attempts."""
+
+    code = "too_many_failed_login_attempts"
+    message = "Too many failed login attempts!"
```

Comment:
Do we handle translation?

Reply:
We never did so far. We can do it in a follow up PR.